### PR TITLE
Add initial inbucket chart

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:
Add the Inbucket which is an email testing application

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
N/A

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/inbucket/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+## Introduction
+
+This chart creates a [Inbucket](https://www.inbucket.org/) deployment on a [Kubernetes](http://kubernetes.io)
+cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.5+ with Beta APIs enabled
```
Comment (unguiculus) [inline incubator/inbucket/README.md]: 1.8+?

File: incubator/inbucket/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta1
```
Comment (unguiculus) [inline incubator/inbucket/templates/deployment.yaml]: I'd suggest `apps/v1beta2`

File: incubator/inbucket/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+description: Inbucket is an email testing application
+name: inbucket
+appVersion: 1.2.0
```
Comment (unguiculus) [inline incubator/inbucket/Chart.yaml]: The current release is 2.0.0. Could you update?

File: incubator/inbucket/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - email
+  - testing
+  - disposable webmail
+home: https://www.inbucket.org/
+sources:
+  - https://github.com/jhillyerd/inbucket
+maintainers:
+  - name: cpanato
+    email: ctadeu@gmail.com
+  - name: jhillyerd
```
Comment (unguiculus) [inline incubator/inbucket/Chart.yaml]: E-mail address missing.

Comment (unguiculus) [reply incubator/inbucket/Chart.yaml]: Can you add an icon?

**Review by unguiculus — CHANGES_REQUESTED** [review]
Nice submission. I'd suggest you move the chart to stable directly.

File: incubator/inbucket/README.md (hunk 1 above)
Comment (cpanato) [reply incubator/inbucket/README.md]: done

File: incubator/inbucket/templates/deployment.yaml (hunk 2 above)
Comment (cpanato) [reply incubator/inbucket/templates/deployment.yaml]: done

File: incubator/inbucket/Chart.yaml (hunk 3 above)
Comment (cpanato) [reply incubator/inbucket/Chart.yaml]: done

File: incubator/inbucket/Chart.yaml (hunk 4 above)
Comment (cpanato) [reply incubator/inbucket/Chart.yaml]: email added. there is no icon until now. I will check with the onwer of the repo 😄

**Comment by cpanato** [comment]
@unguiculus thanks so much for your time to review. I made the changes you proposed and move from `incubator` to `stable`

**Review by unguiculus — CHANGES_REQUESTED** [review]
You only updated the version in `Chart.yaml` but didn't update the Docker image or whatever else is necessary to 2.0.0

**Comment by cpanato** [comment]
@unguiculus the appVersion we are using in this chart is: `1.2.0` and I set the chart version as `1.0.0` because this is the first iteration 😄 

for the app version `2.0.0` I need to check first, there are some changes in the new release.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**